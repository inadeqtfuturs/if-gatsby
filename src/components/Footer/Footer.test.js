import React from 'react';
import { Footer } from '@components';
import ThemeRender from '@utils/testing';

describe('Header', () => {
  it('renders correctly', () => {
    const tree = ThemeRender(
      <Footer>
        <p>hello world</p>
      </Footer>
    );
    expect(tree).toMatchSnapshot();
  });
});
