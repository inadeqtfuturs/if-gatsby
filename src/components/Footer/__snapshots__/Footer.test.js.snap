// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<footer
  className="sc-fzoLag gqDFwP"
>
  <div
    className="sc-AxirZ jpbKm"
    height="100%"
    width={
      Array [
        null,
        "536px",
        "708px",
        "900px",
        "1080px",
        "1200px",
      ]
    }
  >
    <p>
      hello world
    </p>
  </div>
</footer>
`;
